### PR TITLE
Fix #609 Add Captcha translate

### LINKED ISSUE #609: Untranslatable Mage_Captcha module
Sorry for not having time for a PR .... just found this on MSE ...

https://magento.stackexchange.com/questions/236088/magento-translation-not-working/263334#263334

Checked ... seems captcha module never was translatable.

### PR DESCRIPTION
**Pull request by seansan**
https://github.com/OpenMage/magento-lts/issues/609

### CODE REVIEW

File: app/code/core/Mage/Captcha/etc/config.xml [hunk 1]
```diff
@@ -157,6 +157,15 @@
                 </captcha>
             </updates>
         </layout>
+        <translate>
+            <modules>
+                <captcha>
```
Comment (tmotyl) [inline app/code/core/Mage/Captcha/etc/config.xml]: shouldn't this be `<Mage_Captcha>` ?

Comment (kkrieger85) [reply app/code/core/Mage/Captcha/etc/config.xml]: Yes, you're right. 

Did not noticed this

**Comment by sreichel** [comment]
@seansan Do do want to update?

**Comment by Flyingmana** [comment]
after merge of #733 this should not be needed anymore

**Pull request closed without merging**